test/helper.js

```diff
@@ -86,18 +87,24 @@ helper.withTaskCreator = function(mock, skipping) {
 
 /**
  * Set up PulsePublisher in fake mode, at helper.publisher. Messages are stored
+ * Set up tc-lib-pulse in fake mode, with a publisher at at helper.publisher.
  * in helper.messages.  The `helper.checkNextMessage` function allows asserting the
+ * Messages are stored in helper.messages.  The `helper.checkNextMessage`
  * content of the next message, and `helper.checkNoNextMessage` is an assertion that
+ * function allows asserting the content of the next message, and
  * no such message is in the queue.
+ * `helper.checkNoNextMessage` is an assertion that no such message is in the
+ * queue.
```

Comment:
I'm not sure what happened here - maybe a bug in my `git rebase`?  On the assumption it was my fault, I'll fix it up.